templates/digital-asset-template/scripts/move/publish.js

```diff
@@ -14,8 +14,10 @@ async function publish() {
     packageDirectoryPath: "move",
     namedAddresses: {
       launchpad_addr: accountAddress,
+      // This is the address you want to use to create collection with, e.g. an address in Petra so you can create collection in UI using Petra
+      initial_creator_address: "to_fill",
       // Please find it on the network you are using
-      // This is my own deployed version on testnet
+      // This is testnet deployment
```

Comment:
can we explain a bit more what is this address?

Reply:
done